Source/Core/Common/Logging/Log.h

```diff
@@ -76,13 +78,17 @@ enum LOG_LEVELS
 static const char LOG_LEVEL_TO_CHAR[7] = "-NEWID";
 
 void GenericLogFmtImpl(LOG_LEVELS level, LOG_TYPE type, const char* file, int line,
-                       std::string_view format, const fmt::format_args& args);
+                       fmt::string_view format, const fmt::format_args& args);
```

Comment:
Whats the difference between `std::string_view` and `fmt::string_view`? AFAICT `fmt::string_view` is from a time where `std::string_view` wasn't part of the standard yet, and using `std::string_view` should be fine.

Reply:
This doesn't work with FMT_STRING, which returns custom objects that only have a conversion operator for fmt::string_view.